### PR TITLE
Issue #4528: Duplicated fields on comment reply page

### LINKED ISSUE #4528: Comment Body field duplicated on reply page
Was experiencing this problem on a new site, and have replicated the issue on a new local installation running Backdrop 1.16.2 and PHP 7.2.2.

The Comment Body field is duplicated on the comment reply page. On the normal node page with comments at the bottom, the field is printed once. However it's when you press the reply button and get taken to the reply page is when the  Comment Body field is duplicated.

### PR DESCRIPTION
**Pull request by indigoxela**
Fixes https://github.com/backdrop/backdrop-issues/issues/4528

### CODE REVIEW

File: core/modules/comment/tests/comment.test [hunk 1]
```diff
@@ -398,6 +398,7 @@ class CommentInterfaceTest extends CommentHelperCase {
     $this->backdropGet('comment/reply/' . $this->node->nid . '/' . $comment->id);
     $this->assertText($subject_text, 'Individual comment-reply subject found.');
     $this->assertText($comment_text, 'Individual comment-reply body found.');
+    $this->assertUniqueText($comment_text, 'Comment-reply body found only once.');
```
Comment (klonos) [inline core/modules/comment/tests/comment.test:401]: Nice 👍

**Review by klonos — APPROVED** [review]
LGTM 👍

**Comment by indigoxela** [comment]
Requesting fresh sandbox.

**Comment by indigoxela** [comment]
Requesting fresh sandbox, second try.

**Comment by ghost** [comment]
Thanks! Merged into 1.x & 1.16.x.

**Pull request merged**